Automattic/sensei
# Add quiz back to lesson block

Fixes #4372

### Changes proposed in this Pull Request

* Includes Back To Lesson block to be used in the quiz.
* Update blocks initialization to initialize only when necessary.

### Testing instructions


* Enable the feature flag: `add_filter( 'sensei_feature_flag_course_theme', '__return_true' );`
* Create a course with a lesson and a quiz.
* Select the "Sensei Theme" as the "Course Theme" (sidebar).
* Go to the quiz in the frontend, and make sure it contains a "Back to lesson" link.
* Click on the link, and make sure you go to the correct lesson page.


### Screenshot / Video

<img width="551" alt="Screen Shot 2021-11-19 at 18 53 45" src="https://user-images.githubusercontent.com/876340/142696318-cb7323a0-452e-41b7-b061-e78674ad249a.png">
